lang/de_de.lang

```diff
@@ -40,8 +40,7 @@ redactor2_profiles_label_linkify_option_true = Ja
 redactor2_profiles_label_linkify_option_false = Nein
 redactor2_profiles_label_imagetag = HTML-Tag um Bilder<br><i>z.B. p oder figure</i>
 redactor2_profiles_label_redactorplugins = Plugins
-redactor2_profiles_label_redactorcustomplugins = Custom Plugins
-redactor2_profiles_label_redactorcustomplugins_help = Kommagetrennt mit der Syntax "name:pfad/zum/plugin.js"
+redactor2_profiles_label_redactorcustomplugins_help = Bitte eigene Plugins über das ProjektAddon einbinden. (-EP PACKAGES_INCLUDED LATE-)
```

Comment:
`(-EP PACKAGES_INCLUDED LATE-)` was soll dieser hinweis bedeuten?

Reply:
das man seine redactor Plugins über diesen EP und mit Status LATE einbinden muss, damit diese verwendet werden können..

Reply:
vllt sollte man das in einem satz formulieren. die notation hier wirkt so als könnte man sie nur lesen, wenn man weiß was ein EP ist und wenn man weiß das PACKAGES_INCLUDED ein EP ist und dass EPs auch LATE sein können

(impliziert viel detailliertes wissen, was viele nicht haben)

Reply:
ist bereit umformuliert.